shell/start.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+while [ -n "$1" ]
+do
+case "$1" in
+--job) job="$2"
+shift ;;
+--load_from) load_from="$2"
+shift ;;
+--save_to) save_to="$2"
+shift ;;
+--period_start) period_start="$2"
+shift ;;
+--period_end) period_end="$2"
+shift ;;
+*) echo "$1 is not a valid option";;
+esac
+shift
+done
+echo "Start build project..."
+(cd ../ && sbt assembly)
+echo "Build done."
+echo "MongoDB startup..."
+docker run -d -p 27017:27017 mongo:3.6
```

Comment:
Do we need `docker pull mongo:3.6` for the first launch? Please, add this to the script